doc/getting-started.md

```diff
@@ -9,14 +9,15 @@ In this text, we will call the deep neural network that is used to represent the
 3. [Freeze a model](#freeze-a-model)
 4. [Test a model](#test-a-model)
 5. [Compress a model](#compress-a-model)
-6. [Model inference](#model-inference)
+6. [Calculate model deviation](#calculate-model-deviation)
```

Comment:
I would recommend 

1. merge the command line part with "test a model", because the model deviation is a method to test the quality of the model without labels. 
2. merge the python part with "Model inference/Python interface"

What do you think?

Reply:
Agree